sql/updates/world/3.3.5/2020_08_xx_xx_world.sql

```diff
@@ -0,0 +1,15 @@
+-- Disable movement flag updates for Grauf and let script handle them
+UPDATE `creature_template` SET `flags_extra` = `flags_extra`|512 WHERE `entry`=26893;
+-- Disable movement flag updates for Blood-Queen Lana'thel
+UPDATE `creature_template_movement` SET `Ground`= 1, `Flight`= 0 WHERE `CreatureId` IN (37955, 38434, 38435, 38436);
+UPDATE `creature_template` SET `flags_extra`= `flags_extra` | 513 WHERE `entry` IN (37955, 38434, 38435, 38436);
```

Comment:
Should this really be `513` and not `512`?

Reply:
it had 1 before so with that flag =513, not all have 1=INSTANCE_BIND

Reply:
You are OR-ing the value. `flags_extra | 512` means take the existing value and "add" 512. The end-result will be 513 if that flag was already set. -> Change those 513 values to 512 (or even better 0x200).